client/src/components/Learning/Learning.js

```diff
@@ -29,6 +29,8 @@ export default function Learning() {
     setScore(0);
     setAttempted(0);
     setIsSubmitted(false);
+
+    window.scrollTo(0, 0);
```

Comment:
Because when I click on the link to open a new page, it show the top part of the new page. The scroll stays where it was on the previous page.

I hope that answers your question. For any other confusion, feel free to reach out to me via chats ☺️

Reply:
Okay. 👍🏻